app/views/rubygems/_rubygem.html.erb

```diff
@@ -1,15 +1,25 @@
 <%= link_to rubygem_path(rubygem.name), :class => 'gems__gem' do %>
-  <span class="gems__gem__info">
+  <% if yanked?(rubygem) %>
+    <span class="gems__gem__info">
+    <h2 class="gems__gem__name">
+      <%= rubygem.name %>
+      <span class="gems__gem__badge">Yanked</span>
+    </h2>
+    <p class="gems__gem__desc t-text"><%= t('rubygems.show_yanked.not_hosted_notice') %></p>
+  </span>
+  <% else %>
+    <span class="gems__gem__info">
     <h2 class="gems__gem__name">
       <%= rubygem.name %>
       <span class="gems__gem__version"><%= latest_version_number(rubygem) %></span>
     </h2>
     <p class="gems__gem__desc t-text"><%= short_info(rubygem) %></p>
   </span>
-  <p class="gems__gem__downloads__count">
-    <%= download_count rubygem %>
-    <span class="gems__gem__downloads__heading">
+    <p class="gems__gem__downloads__count">
+      <%= download_count rubygem %>
+      <span class="gems__gem__downloads__heading">
       <%= t('rubygems.index.downloads') %>
     </span>
-  </p>
+    </p>
```

Comment:
Please fix indentation of [these lines](https://github.com/rubygems/rubygems.org/blob/9720ac7932faa36b27bf22c5799955a82e2c52a9/app/views/rubygems/_rubygem.html.erb#L11-L23)

Reply:
I added an if statement so, this intendation was added.

Reply:
check the link i have mentioned. to be specific, make sure span opening and closing tags are alligned, h2 is indented within span and bottom span content is indented.